app/src/main/java/net/aiscope/gdd_app/ui/mask/customview/MaskCustomView.kt

```diff
@@ -134,6 +145,8 @@ class MaskCustomView @JvmOverloads constructor(
 
     fun redoAvailable() = maskLayer.redoAvailable()
 
-    class MaskCustomViewSavedState(superState: Parcelable?, val maskLayerState: Parcelable) :
-        BaseSavedState(superState)
+    class MaskCustomViewSavedState(
+        superState: Parcelable?,
+        val maskLayerControllerState: Parcelable
```

Comment:
ups, I guess that this is an artefact of the double rename, would it make sense to rename this back as well?

Reply:
Sure, thanks for noticing!